content/quizzes/en.yaml

```diff
@@ -4092,13 +4092,13 @@ levels:
             code: |-
                 name = {ask} 'What is your name?'
                 {if} name == 'Hedy':
-                password = {ask} 'What is your password?'
+                    password = {ask} 'What is your password?'
                     {if} password =='turtle123':
                         {print} 'Yey'
                     {else}:
                         {print} 'Access denied'
                 {else}:
-                 {print} 'Go fish'
```

Comment:
What I know is that only 1 Option can be the right choice. Without the space it is definitely wrong. With 1 space, the indentation is not consistent, but it will run and execute fine. So to remove all doubt, better to have no space at all.

Reply:
Oh no, it should not run and execute fine. Or at least this is not the intent since there is an explicit check for inconsistent indentation. When I execute the snippet in level 17, it does generate an error (not a great one but it fails for sure):
 
<img width="748" alt="Screenshot 2024-04-29 at 15 32 07" src="https://github.com/hedyorg/hedy/assets/3010723/283705f7-f5b6-4936-957a-307a4f01dbb6">

Can you please point out when it executes fine?

Reply:
Ah OK, I was thinking of Python, but I guess at level 17, this is an error. So you're right, probably best to leave it in!